package.json

```diff
@@ -14,8 +14,10 @@
     "lint:fix": "npm-run-all eslint:fix",
     "perf": "cd ./examples/perf && npm run test:ci",
     "prepublish": "npm-run-all clean build",
-    "publish": "lerna publish --conventional-commits",
-    "publish:beta": "lerna publish --conventional-commits --npm-tag=next --preid=next",
+    "version": "lerna publish --conventional-commits --changelog-preset=angular-all --skip-npm",
```

Comment:
@azu, this is a matter of taste, but how about `version:up` or `version:increment`? Since `version:cancel` is there, the `version` in this context is a noun. It is natural for me to use a verb in `npm-run-script`. (Also `yarn lerna version` is sometimes confusing with `yarn lerna -v`?)

Reply:
I've rewritten script complety

1. `prepare-release`
2. `release`

Reply:
Oh my. Looks really great to me!